README.rst

```diff
@@ -336,6 +336,9 @@ List of supported endpoints
     amadeus.analytics.itinerary_price_metrics.get(originIataCode='MAD', destinationIataCode='CDG',
                                                 departureDate='2021-03-21')
 
+    # Travel Restrictions
+    amadeus.travel_restrictions.covid19_area_report.get(countryCode="US")
```

Comment:
The SDK maps every API path to a similar path. For example, `/v1/duty-of-care/diseases/covid19-area-report`
should be represented as`amadeus.duty_of_care.diseases.covid19_area_report`.
So the classes `DutyOfCare` and `Diseases` have to be created and delete the `TravelRestrictions`.

Reply:
Ah, understood, will make those changes.